scripts/cmsclean.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+echo "Warning: about to delete the following directories:
+~/.politeiawww/data/testnet3/
```

Comment:
Everything that is a directory should be a variable so they can be changed externally. Furthermore the path should be made up of different elements. For example
```
TESTNET=testnet3
PIDHOME=~/.politeiad
PIDDATA=${PIDHOME}/data/${TESTNET}
```

With that we can alter the call to the script and make it work on other OS' or alternate installation locations.

The script should either support windows and OSX and terminate or create a conditional in the script for said OS variables to their standard directories.